wagtail/admin/forms/pages.py

```diff
@@ -178,11 +178,16 @@ def is_valid(self):
     def clean(self):
         cleaned_data = super().clean()
         if "slug" in self.cleaned_data:
-            if not Page._slug_is_available(
-                cleaned_data["slug"], self.parent_page, self.instance
-            ):
+            page_slug = cleaned_data["slug"]
+            if not Page._slug_is_available(page_slug, self.parent_page, self.instance):
                 self.add_error(
-                    "slug", forms.ValidationError(_("This slug is already in use"))
+                    "slug",
+                    forms.ValidationError(
+                        _(
+                            "The slug '%(page_slug)s' is already in use in use within the parent page"
```

Comment:
Looks like 'in use' is duplicated here (and in all the resulting test cases).

Reply:
I'll fix it up tomorrow - thanks for noticing Matt.

Reply:
https://github.com/wagtail/wagtail/pull/9718